samples/suit/smp_transfer/README.rst

```diff
@@ -0,0 +1,382 @@
+:orphan:
```

Comment:
Note that this won't be applied in the current structure, because the :orphaned: SUIT samples are made visible by the toctree inclusion on this page: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/suit.html

Both samples visible there are orphaned.